theSrc/R/htmlwidget.R

```diff
@@ -209,6 +209,10 @@
 #' @param y.midpoint.line.color y midpoint line color
 #' @param y.midpoint.line.dash y midpoint line type. Can be one of 'solid', 'dot', 'dash'.
 #' @param y.midpoint.line.width y midpoint line width in pixels
+#' @param top.left.quadrant.color Color of the top left quadrant
```

Comment:
I'd prefer the name `quadrant.top.left.color` so that these parameters start with the same prefix.

Reply:
Good call! Updated in https://github.com/Displayr/rhtmlCombinedScatter/pull/76/commits/9e208a3345157d31f97ad323ffd850f225c17e31